canonical-web-and-design/maas-ui
# Set up actions and state necessary for add machine form.

## Done
- Added actions, reducers and selectors for fetching domains.
- Added actions, reducers and selectors for creating machines.
- Fetched all the new stuff in the AddMachine component.

## QA
- Go to `/r/machines/add` and check that everything you find at `/#/machines > Add hardware > Machine` is fetched.

Fixes canonical-web-and-design/MAAS-squad#1811
